jobs/pacman-awsrules/src/test/java/com/tmobile/cloud/gcprules/GKECluster/Disable_Web_UI.java

```diff
@@ -0,0 +1,104 @@
+package com.tmobile.cloud.gcprules.GKECluster;
```

Comment:
Please rename the class name, use mixed case without underscore with the first letter of each internal word capitalized.